### DISCUSSION TITLE
消息队列销毁问题

### DISCUSSION THREAD

**Discussion by susahdh** [cloudwu/skynet#1826] [General]
static void
drop_message(struct skynet_message *msg, void *ud) {
	struct drop_t *d = ud;
	skynet_free(msg->data);
	uint32_t source = d->handle;
	assert(source);
	// report error to the message source
	skynet_send(NULL, source, msg->source, PTYPE_ERROR, 0, NULL, 0);
}
销毁消息队列直接就释放了msg->data，如果这里带有指针，是会导致内存泄露？

**Comment by cloudwu** [comment]
如果有疑虑，请给出明确的测试案例。
